CONTRIBUTING.md

```diff
@@ -5,9 +5,17 @@
 * Use `use` statements wherever possible instead of writing the fully qualified name.
 * Order the composer/npm dependencies alphabetically.
 * Do not use code from the [includes](./includes) directory anywhere else.
-* Please cover your code by unit tests. Code under `includes` does not require tests.
+* Please cover your code by unit tests, our goal is 100% line coverage. Code under `includes` does not require tests as it's mostly not testable.
 * Do not use vendor prefixes like `-webkit` in styles.
   This is done by PostCSS + Autoprefixer according to the [`.browserslistrc`](./.browserslistrc).
 
 ## Pull requests
-Please create single pull requests for every feature instead of creating one big monster of pull request containing a complete rewrite.
+* The PR should contain a short overview of the changes.
+* Before implementing bigger changes, please open an issue to discuss the feature and possible implementation options.
+* Please create single pull requests for every feature instead of creating one big monster of pull request containing a complete rewrite.
+* Squash similar commits to make the review easier.
```

Comment:
I would just enable squash commit PR merges for the repo and skip this point. In my opinion it's not worth the discussions. If the commits of a PR are not great: just squash it :hammer:

Reply:
That's something only @msquare can do because i sadly don't have the permissions to :(

Reply:
But in general ppl should also just do useful commits and that normally also means squashing

Reply:
All possible merge options are already enabled, can you please check?

Reply:
It is & i guess i was dumb :upside_down_face:

Reply:
Anyhow, i would prefer to have it there so that eventually ppl will just squash their commits as it might be useful to just have two or three instead of ten changing every file by its own